examples/exex/rollup/src/db.rs

```diff
@@ -0,0 +1,482 @@
+use std::{
+    collections::{hash_map::Entry, HashMap},
+    ops::Deref,
+    str::FromStr,
+    sync::{Arc, Mutex, MutexGuard},
+};
+
+use reth::revm::db::{
+    states::{PlainStorageChangeset, PlainStorageRevert},
+    BundleState,
+};
+use reth_primitives::{
+    revm_primitives::{AccountInfo, Bytecode},
+    Address, Bytes, SealedBlockWithSenders, StorageEntry, B256, U256,
+};
+use reth_provider::{bundle_state::StorageRevertsIter, OriginalValuesKnown, ProviderError};
+use reth_tracing::tracing::error;
+use rusqlite::Connection;
+
+/// Type used to initialize revms bundle state.
+type BundleStateInit =
+    HashMap<Address, (Option<AccountInfo>, Option<AccountInfo>, HashMap<B256, (U256, U256)>)>;
+
+/// Types used inside RevertsInit to initialize revms reverts.
+pub type AccountRevertInit = (Option<Option<AccountInfo>>, Vec<StorageEntry>);
```

Comment:
this doesn't look very nice, can we remove those aliases?

Reply:
we surely can, or I can reference the original aliases here https://github.com/paradigmxyz/reth/blob/a8cd1f71a03c773c24659fc28bfed2ba5f2bd97b/crates/storage/provider/src/bundle_state/bundle_state_with_receipts.rs#L37-L45